scripts/ci/build_benchmarks.sh

```diff
@@ -23,12 +23,29 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 #
+set -xeuo pipefail
 
 # Builds CI benchmarks and checks test status
 pushd $GITHUB_WORKSPACE/test/benchmarking && \
-mkdir build && cd build && \
+mkdir -p build && cd build && \
 cmake -DCMAKE_PREFIX_PATH=$GITHUB_WORKSPACE/dist ../src && make && \
 popd
+# man bash
+# ...
+# -e The shell does not exit if the command that fails is part of the command
+# list immediately following a while or until keyword, part of the test 
+# following the if or elif reserved words, part of any command executed in a && 
+# or || list except the command following the final && or  || ...
+# ...
+# given the above, explicitly check and bail.
+poss_ret_exit_code=$?
```

Comment:
I'm not sure why these were originally chained like this, but it seems simpler to just separate the commands out.